resources/public/api/conf/6.0/application.yaml

```diff
@@ -73,7 +73,9 @@ paths:
   /individuals/losses/{nino}/loss-claims/tax-year/{taxYearClaimedFor}:
     $ref: "./loss_claims_list.yaml"
   /individuals/losses/{nino}/loss-claims/{claimId}:
-    $ref: "./loss_claims_claimId_endpoints.yaml"
+    $ref: "./loss_claims_retrieve.yaml"
+  /individuals/losses/{nino}/loss-claims/{claimId}/tax-year/{tax-year}:
```

Comment:
This is the wrong url. Please update to   /individuals/losses/{nino}/loss-claims/{claimId}/tax-year/**{taxYearClaimedFor}**: like in order and list, thanks